### PR TITLE
Add multi hosts

### PR DESCRIPTION
**Pull request by Gerych1984**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #159

1. Allow several hosts in `Route::host` method
2. Add method `Route::isMultiHost`

### CODE REVIEW

File: src/Route.php [hunk 1]
```diff
@@ -186,13 +186,27 @@ public function pattern(string $pattern): self
     /**
      * @return self
      */
-    public function host(string $host): self
+    public function host(string ...$hosts): self
```
Comment (samdark) [inline src/Route.php:189]: That looks a bit weird in routes:

```php
->host('demo.test', 'demo.com')
```

According to English grammar it should be hosts, not host. Maybe we need an alias method?

```suggestion
    public function hosts(string ...$hosts): self
    {
        // call host() here
    }

    public function host(string ...$hosts): self
```

File: src/Route.php [hunk 2]
```diff
@@ -369,7 +390,8 @@ public function __debugInfo()
             'name' => $this->name,
             'methods' => $this->methods,
             'pattern' => $this->pattern,
-            'host' => $this->host,
+            'host' => $this->getData('host'),
+            'hosts' => $this->hosts,
```
Comment (samdark) [inline src/Route.php:394]: Having both host and hosts looks unnecessary.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Looks good. Need a line for CHANGELOG.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 ## 1.0.1 under development
 
-- no changes in this release.
+- Allow multiple separate hosts with new `Route::hosts` method.
```
Comment (rustamwin) [inline CHANGELOG.md:5]:
```suggestion
- Enh #163: Allow multiple separate hosts with new `Route::hosts` method.
```

**Comment by samdark** [comment]
Thanks!

**Pull request merged**